packages/datadog-plugin-grpc/src/client.js

```diff
@@ -29,6 +30,11 @@ class GrpcClientPlugin extends ClientPlugin {
       }
     })
 
+    // needed as precursor for peer.service
+    if (method.service && method.package) {
+      span.setTag('rpc.service', method.package + '.' + method.service)
```

Comment:
Another way of thinking of this is if the values are absent, is it better to have `rpc.service` tag values of `.foo` / `bar.` / `.` as opposed to `MyPackage.foo` / `bar.MyService` / `MyPackage.MyService`?

It's up to you. The empty values would certainly indicate an issue though it might not be clear where the issue originates. The default `My*` values would certainly make it easier to grep where an issue is coming from.

```js
`${method.package||'MyPackage'}.${method.service||'MyService'}`
```

Reply:
in this case we also have the other tags that will give us some hints about which information is missing
* `grpc.package`
* `grpc.path`

For peer.service, setting defaults like MyService will result in having service maps that is not coherent so I prefer not to infer a service when the information are not fully known

Reply:
With help from @rarguelloF we determined that Python had unrealistic test scenarios that weren't using generated stubs, but low-level API in which that's a possibility. We're all good here.